crates/nu-command/src/filters/flatten.rs

```diff
@@ -157,7 +157,7 @@ fn flat_value(columns: &[CellPath], item: Value, all: bool) -> Vec<Value> {
             let mut inner_table = None;
 
             for (column_index, (column, value)) in val.into_owned().into_iter().enumerate() {
-                let column_requested = columns.iter().find(|c| c.to_string() == column);
+                let column_requested = columns.iter().find(|c| c.to_column_name() == column);
```

Comment:
Is `flatten` accepting a nested cell path semantically or only a single column level?

Reply:
It only accepts a single (string) column name, and trying to pass it a cell-path results in an error.

Reply:
Ah OK, otherwise we could have moved away from using `CellPath` internally to something more explicit as just a string.